test_scripts/Polices/External_UCS_Informing_HMI/002_ATF_P_GetListofPermissions_with_appID_all_ecs_lower_bound_EntitySatus_ON.lua

```diff
@@ -50,9 +50,26 @@ end
 function Test:Precondition_PTU_and_OnAppPermissionConsent_AllParams_Lower()
   local ptu_file_path = "files/jsons/Policies/Related_HMI_API/"
   local ptu_file = "OnAppPermissionConsent_ptu.json"
+  local time_onAppPermissionChanged = 0
 
   testCasesForPolicyTable:flow_SUCCEESS_EXTERNAL_PROPRIETARY(self, nil, nil, nil, ptu_file_path, nil, ptu_file)
```

Comment:
ptu function should be placed after expectation EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged")
Let's consider simple case
line1: sendRPC(RegisterAppInterface)
line2: ExpectResponse(RegisterAppInterface)
It's works fine because SDL need some time to process incoming request, it validates parameters. constructs response,
so ATF have enough time to start the expectation. But in case of limited resources there could be unpredicted behavior, when ATF in ordered loop could get cpu laterand expectation could not work.
in this simple case it is absolutely impossible situation even if PC is overloaded, but let's consider case from this script.
line1: function ()
line2: cal local fucnction()
line3: EXPECT(smth)
line4: end

Local functions do some action that raises some SDL's beahavior and here it is very bad to separate function and expectation. It is good if local function is just send smth to SDL and that's all, but if there is some expectation and another one is outside, the sporadic behavior is garanteed